v20.2/performance.md

```diff
@@ -15,44 +15,52 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
 
 ## Scale
 
-TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can process **1.2M tpmC with 100,000 warehouses, a nearly perfect score.** For a refresher on what exactly TPC-C is and how it is measured, see [Benchmarks used](#benchmarks-used) below.
+TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can process **1.68M tpmC with 140,000 warehouses, a nearly perfect score.** As shown in the chart below, this is a 42% improvement in efficiency over the results from CockroachDB 19.2.
 
-<img src="{{ 'images/v20.2/tpcc100k.png' | relative_url }}" alt="TPC-C 100,000" style="max-width:100%" />
+For a refresher on what exactly TPC-C is and how it is measured, see [Benchmarks used](#benchmarks-used) below.
 
-Metric                             | CockroachDB Results       
------------------------------------|-------------------
-Max Throughput                     | 1,245,462 tpmC    
-Max Warehouses with Max Efficiency | 100,000 Warehouses
-Max Number of Rows                 | 49.8B             
-Max Unreplicated Data              | 8TB               
-Number of Nodes                    | 81
-Node Machine Type                  | c5d.9xlarge            
+(<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Replace this chart with the one provided by Design)
 
+<img src="{{ 'images/v20.2/tpcc140k.png' | relative_url }}" alt="TPC-C 140,000" style="max-width:100%" />
+
+| Metric                             | CockroachDB Results |
+|------------------------------------+---------------------|
+| Max Throughput                     | 1,684,437 tpmC      |
+| Max Warehouses with Max Efficiency | 140,000 Warehouses  |
+| Max Number of Rows                 | 49.8B               |
```

Comment:
I suspect many of the values in this table need to be edited.  I took a shot at `Max Throughput` and `Max Warehouses`, but not sure if they are correct.  The others I didn't touch, but they're probably stale now (e.g. data, rows)